app/repositories/XiEoriInformationRepository.scala

```diff
@@ -43,7 +44,7 @@ class DefaultXiEoriInformationRepository @Inject()(
       IndexModel(
         ascending("lastUpdated"),
         IndexOptions().name("xieori-information-last-updated-index")
-          .expireAfter(config.get[Int]("mongodb.timeToLiveInHoursXieoriInformation"), TimeUnit.HOURS)
+          .expireAfter(config.get[Long]("mongodb.timeToLiveInHoursXieoriInformation"), TimeUnit.HOURS)
```

Comment:
Because if I don't it will throw this error at all the place I have updated this.

[error] -- [E007] Type Mismatch Error: /CDS/customs-data-store/app/repositories/XiEoriInformationRepository.scala:47:23
[error] 47 |          .expireAfter(config.get[Int]("mongodb.timeToLiveInHoursXieoriInformation"), TimeUnit.HOURS)
[error]    |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[error]    |                       Found:    Int
[error]    |                       Required: Long

Reply:
Ok